homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -375,19 +375,25 @@ def _handle_none_keypad_serial(keypad_device: dict, bridge_serial: int) -> str:
     return keypad_device["serial"] or f"{bridge_serial}_{keypad_device['device_id']}"
 
 
-def _area_name_from_id(areas: dict[str, dict], area_id: str) -> str:
+def _area_name_from_id(
+    areas: dict[str, dict], area_id: str, previous_area_part: str = ""
```

Comment:
```suggestion
    areas: dict[str, dict], area_id: str | None, previous_area_part: str = ""
```

This has to be able to be `None` or the `if area_id is None:` would never be possible